src/exec/builtins/args.rs

```diff
@@ -30,6 +30,12 @@ pub struct ChangeDirectoryArgs {
 pub struct ListDirectoryArgs {
     #[arg(short = 'a', long = "all", help = "Show hidden files and directories")]
     pub show_hidden: bool,
+    #[arg(short = 'l', long = "long", help = "Show files with details as a table")]
+    pub long_view: bool,
+    #[arg(short = 'o', long = "octal-permissions", help = "Use octal permissions instead of string representation")]
+    pub octal_permissions: bool,
+    #[arg(long = "permission-seperator", help = "Show sperators for each permission group (eg. Current user, current group, and other users and groups)")]
+    pub permission_seperator: bool,
```

Comment:
Clap can infer the long form of some of these parameters so you can probably remove the `= "..."` on `octal_permissions` and `permissions_seperator`

Reply:
Fixed via b7f738c

Reply:
I think its nice for readability, but I can change it if you *really* want me to